packages/eslint-config-qlean/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-qlean",
+  "version": "1.0.0",
+  "description": "JavaScript Qlean Style - EsLint Shareable Config",
+  "main": "index.js",
```

Comment:
Do we even have this file?

Reply:
@alextewpin yeah, shareable config is there